src/redux/wallets/actions.ts

```diff
@@ -514,13 +518,6 @@ export const sendTransferTransaction = (
         });
 
         if (appWallet.type === WalletType.HW) {
-            dispatch({
```

Comment:
why did you remove this one ?

I think it would be good to display these messages:
1. connecting to ledger
2. make sure app is opened
3. review tx
4. broadcasting